python/packages/autogen-ext/src/autogen_ext/agents/video_surfer/_video_surfer.py

```diff
@@ -0,0 +1,29 @@
+from autogen_agentchat.agents import AssistantAgent
+
+from ._action_space import extract_audio, get_video_length, transcribe_audio_with_timestamps, get_screenshot_at, save_screenshot, openai_transcribe_video_screenshot, download_youtube_video
+
+
+class VideoSurferAgent(AssistantAgent):
```

Comment:
Use class doc string for the documentation instead of the README.md. Much easier to discover.